decompiler/__init__.py

```diff
@@ -261,21 +261,21 @@ def print_with(self, ast):
             # Sanity check. check if there's a matching with statement two nodes further
             if not(isinstance(self.block[self.index + 2], renpy.ast.With) and
                    self.block[self.index + 2].expr == ast.paired):
-                raise Exception("Unmatched paired with {0} != {1}".format(
-                                repr(self.paired_with), repr(ast.expr)))
+                raise Exception(
+                    f'Unmatched paired with {repr(self.paired_with)} != {repr(ast.expr)}')
```

Comment:
You can use the !r conversion here for both items, as in `{self.paired_with!r}`

Reply:
Done